config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -9,50 +9,113 @@ spec:
   nodeselector: {}
   bytecode:
     image:
-      url: quay.io/bpfman-bytecode/go-app-counter:latest
+      url: quay.io/bpfman-bytecode/app-test:latest
+      # imagepullpolicy: Always
   programs:
-    - type: Kprobe
+    - bpffunctionname: kprobe_test
+      type: Kprobe
       kprobe:
-        bpffunctionname: kprobe_counter
-        func_name: try_to_wake_up
-        offset: 0
-        retprobe: false
-    - type: Tracepoint
+        attach_points:
```

Comment:
👀 inconsistent rules re: casing. k8s CRDs are supposed to be camelCase

Reply:
Is the JSON representation supposed to be camelCase?